### PR TITLE
Added GNU standard version option.

### PR DESCRIPTION
**Pull request by cacciatc**
PR to address https://github.com/KnightOS/KnightOS/issues/353

This PR includes a bash script that generates a version header file. This header file includes a license, copyright, and version string built using git describe. The current build process from the developer's viewpoint does not change.

Example output:

```
bin/kpack -v
kpack (KnightOS) 1.0.10
Copyright (c) 2014 The KnightOS Group
License MIT/X11
```

### CODE REVIEW

**Comment by SirCmpwn** [comment]
Thanks!

File: src/version.h [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#ifndef VERSION_H
+#define VERSION_H
+#define GIT_SHORT_VERSION ("1.0.10-dirty")
+#define COPYRIGHT ("Copyright (c) 2014 The KnightOS Group\n")
```
Comment (MaxLeiter) [inline src/version.h]: 2016?

Comment (SirCmpwn) [inline src/version.h]: Lol, how about just making it $current_year

**Comment by cacciatc** [comment]
Just submitted a PR fix. Yeah, I just pulled the copyright info from somewhere else in the repo :)

**Pull request merged**